sql/core/src/main/scala/org/apache/spark/sql/execution/basicPhysicalOperators.scala

```diff
@@ -77,9 +77,40 @@ case class ProjectExec(projectList: Seq[NamedExpression], child: SparkPlan)
     }
   }
 
-  override def outputOrdering: Seq[SortOrder] = child.outputOrdering
+  override def outputOrdering: Seq[SortOrder] =
+    ProjectHelper.outputOrdering(projectList, child.outputOrdering, child)
 }
 
+object ProjectHelper {
+  /**
+   * Determins the outputOrdering property for [[ProjectExec]] and [[TakeOrderedAndProjectExec]]
```

Comment:
@tejasapatil Thanks. I just think having a meaningless sort order looks weird. Since it has no harm, I closed this.